### PR TITLE
Migrate to GitHub Actions

### PR DESCRIPTION
**Pull request by ihostage**

### CODE REVIEW

**Comment by lightbend-cla-validator** [comment]
At least one pull request committer is not linked to a user. See https://help.github.com/en/articles/why-are-my-commits-linked-to-the-wrong-user#commits-are-not-linked-to-any-user

File: .github/workflows/publish.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    name: Publish / Artifacts
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          # we don't know what commit the last tag was it's safer to get entire repo so previousStableVersion resolves
+          fetch-depth: 0
+
+      - name: Set up JDK 8
```
Comment (octonato) [inline .github/workflows/publish.yml:20]: Yep, let's never forget to release with jdk8. 😆

**Review by octonato — APPROVED** [review]
LGTM

File: .github/workflows/publish.yml (hunk 1 above)
Comment (ihostage) [reply .github/workflows/publish.yml:20]: Yes, you know that better than me 😂

**Pull request merged**